experiments/AMIP/modular/components/ocean/prescr_seaice_init.jl

```diff
@@ -2,6 +2,8 @@ import ClimaTimeSteppers as CTS
 import ClimaCoupler.Interfacer: SeaIceModelSimulation, get_field, update_field!, name
 import ClimaCoupler.FieldExchanger: step!, reinit!
 import ClimaCoupler.FluxCalculator: update_turbulent_fluxes_point!
+using ClimaCoupler: Regridder
```

Comment:
Again, checking that this is needed?

Reply:
Used [here](https://github.com/CliMA/ClimaCoupler.jl/blob/115f3c37798ac368df1ae2e2eb9691badd630432/experiments/AMIP/modular/components/ocean/prescr_seaice_init.jl#L81)